app/assets/javascripts/components/overview/available_actions.jsx

```diff
@@ -72,7 +72,7 @@ const AvailableActions = createReactClass({
     }
 
     const enteredTitle = prompt(I18n.t('courses.confirm_course_deletion', { title: this.props.course.title }));
-    if (enteredTitle.trim() === this.props.course.title.trim()) {
+    if (enteredTitle !== null && enteredTitle.trim() === this.props.course.title.trim()) {
```

Comment:
Why is this necessary? Are there cases where the trimmed title from the props could be null?

Reply:
it is not visible in prod but when i cancel the prompt while it is empty in dev it shows an error.

[Screencast from 14-02-24 12:28:01 PM IST.webm](https://github.com/WikiEducationFoundation/WikiEduDashboard/assets/92005669/d5f34d71-d5f1-498c-a1c9-0f04c3670494)

Reply:
Cool, that makes sense. Please add a comment noting what situations these guard checks are for.